src/index.js

```diff
@@ -195,6 +197,25 @@ export default class ToggleBlock {
     }
   }
 
+  /**
+   * With the key combination shift + tab the toggle block is extracted.
+   *
+   * @param {block} item
+   * @param {KeyboardEvent} e
+  */
+  extractionBlock(item, e) {
+    if (e.code === 'Tab' && e.shiftKey) {
+      const indexBlock = this.api.blocks.getCurrentBlockIndex();
+
+      const children = document.querySelectorAll(`div[foreignKey="${this.wrapper.id}"]`);
+      const length = children.length;
```

Comment:
@juliansg022 I can't use destructuring in this case, because the length variable would be `NaN`, what do you think is the problem?

Reply:
@Jenietoc `length` should be at least `1`, and there shouldn't be a problem because `children` always will have elements and therefore the `length` always will have a defined value.